spyder/config/main.py

```diff
@@ -93,6 +93,8 @@
               'cpu_usage/timeout': 2000,
               'use_custom_margin': True,
               'custom_margin': 0,
+              'use_custom_cursor_blinking': False,
+              'custom_cursor_blinking': 1000,

```

Comment:
You are right. This makes sense to set the default value here as the default value for the OS. I will change it.